### PR TITLE
Updated README.md with standard Markdown tables, cleaned a bit header…

### PR DESCRIPTION
**Pull request by Maxzor**
… levels.
#90

### CODE REVIEW

File: python_smi_tools/rocm_smi.py [hunk 1]
```diff
@@ -188,7 +188,7 @@ def getFanSpeed(device):
     if rsmi_ret_ok(ret, device, None, True):
         fm = fanMax.value
     if fl == 0 or fm == 0:
-        return (fl, fm) #  to prevent division by zero crash
+        return (fl, 0)  # to prevent division by zero crash
```
Comment (keryell) [inline python_smi_tools/rocm_smi.py:191]: This looks outside the scope of a documentation change.

**Review by keryell — COMMENTED** [review]
That looks quite nicer!
Thanks.

**Comment by dmitrii-galantsev** [comment]
looks like I broke something when trying to rebase using github GUI :)
I'll cherry-pick your change and apply myself. Should be in https://github.com/RadeonOpenCompute/rocm_smi_lib/tree/develop soon

**Comment by dmitrii-galantsev** [comment]
Merged 8cfcb51550b2a2c1a406465a6b8d994a13cf0f9e
